File: Program/ActEditor/src/pl/wroc/pwr/student/acteditor/test/Tester.java
package pl.wroc.pwr.student.acteditor.test;

import org.eclipse.swt.layout.FormLayout;
import org.eclipse.swt.widgets.Display;
import org.eclipse.swt.widgets.Shell;

public class Tester {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		final Display display = new Display();
		Shell shell = new Shell(display);
		shell.setSize(715, 555);
		shell.setText("Hello world!");
		shell.setLayout(new FormLayout());
		shell.setMaximized(true);
		shell.open();
		while (!shell.isDisposed()) {
			if (!display.readAndDispatch())
				display.sleep();
		}
		display.dispose();
	}
}

File: Program/ActEditor/bin/pl/wroc/pwr/student/acteditor/test/Tester.js
Clazz.declarePackage ("pl.wroc.pwr.student.acteditor.test");
Clazz.load (null, "pl.wroc.pwr.student.acteditor.test.Tester", ["$wt.layout.FormLayout", "$wt.widgets.Display", "$.Shell"], function () {
c$ = Clazz.declareType (pl.wroc.pwr.student.acteditor.test, "Tester");
c$.main = Clazz.defineMethod (c$, "main", 
function (args) {
var display =  new $wt.widgets.Display ();
var shell =  new $wt.widgets.Shell (display);
shell.setSize (715, 555);
shell.setText ("Hello world!");
shell.setLayout ( new $wt.layout.FormLayout ());
shell.setMaximized (true);
shell.open ();
Sync2Async.block (shell, this, function () {
display.dispose ();
});
return;
}, "~A");
});
